.github/workflows/Documentation.yml

```
@@ -2,17 +2,22 @@ name: Documentation
 
 on:
   push:
-    branches:
-      - 'master'
     tags: '*'
   pull_request:
 jobs:
   Documenter:
     name: Documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: julia-actions/julia-buildpkg@latest
+      - uses: actions/checkout@v3
+      - name: "Monorepo setup"
+        # also dev OndaEDFSchemas.jl for pull request builds
+        if: ${{ github.event_name == 'pull_request' }}
+        shell: julia --color=yes --project=docs {0}
+        run: |
+          using Pkg
+          Pkg.develop([PackageSpec(; path="./OndaEDFSchemas.jl"),
+                       PackageSpec(; path=".")])
```

I think you want to separate these two package dev calls now:
```suggestion
          Pkg.develop(PackageSpec(; path="./OndaEDFSchemas.jl"))
      - name: "Use latest OndaEDF.jl"
        shell: julia --color=yes --project=docs {0}
        run: |
          using Pkg
          Pkg.develop(PackageSpec(; path="."))
```